.npmrc

```diff
@@ -0,0 +1,4 @@
+@stage-tech:registry=https://npm.pkg.github.com/stage-tech
```

Comment:
Add his file to gitignore

#npm local config
.npmrc

Reply:
changed